classes/ValidFormBuilder/ValidForm.php

```diff
@@ -1847,13 +1847,16 @@ protected function __toJS($strCustomJs = "", $arrInitArguments = array(), $blnRa
      * @param $intDynamicCount
      * @return bool|true
      */
-    protected function elementShouldDisplay(Base $objElement, $intDynamicCount = 0)
+    protected function elementShouldDisplay(Base $objElement)
     {
         $blnReturn = true;
 
         $objCondition = $objElement->getConditionRecursive("visible");
         if (is_object($objCondition)) {
-            if ($objCondition->isMet($intDynamicCount)) {
+            // If the condition is met for the first element,
+            // it automatically applies to the dynamic clones as well.
+            // Therefore we only check dynamic counter 0
+            if ($objCondition->isMet(0)) {
```

Comment:
No need to hard code the 0. It's already set via the default value of $intDynamicCount.